quantmsio/operate/tools.py

```
@@ -0,0 +1,246 @@
+                x["scan"],
+                mzml_handlers,
+                mzml_directory,
+            ),
+            axis=1,
+            result_type="expand",
+        )
+        pqwriters, pqwriter_no_part = save_parquet_file(partitions, table, output_folder, filename, pqwriters, pqwriter_no_part, PSM_SCHEMA)
+    close_file(partitions, pqwriters, pqwriter_no_part)
+
+
+def save_parquet_file(partitions, table, output_folder, filename, pqwriters = {}, pqwriter_no_part=None, schema=FEATURE_SCHEMA):
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)